spec/features/user_creates_review_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+# Acceptance Criteria
+# [ ] I want to see a button to create my account from the index page
+# [ ] I want to visit a create account page when I click on this button
+# [ ] I want to fill in my account details (name, username, email, and password)
+# [ ] I want to get error messages if I do not fill out all required fields
+# [ ] Once I have created an account, I want to be logged in
+
+feature "user creates review" do
+
+  before(:each) do
+    @user = FactoryGirl.create(:user)
+    login_as(@user, scope: :user)
+  end
+
```

Comment:
You create the same show for each of your scenario blocks. As a result, you should actually create it at the top of your test up here with your user.

I would also strongly suggest you use the "let!() syntax over the "before...each...do" syntax, but its not required!